iOS/Flows/Threads/ThreadViewController.swift

```diff
@@ -77,7 +77,7 @@ class ThreadViewController: DiffableCollectionViewController<MessageSequenceSect
     }
 
     override var canBecomeFirstResponder: Bool {
-        return true
+        return self.presentedViewController.isNil
```

Comment:
Thread VC was becoming first responder again while the emoji picker was up. When we told it to become first responder again, it does nothing because it thinks it's already first responder.